rt-PsychoPy/rt-network_feedback.py

```diff
@@ -254,13 +305,18 @@
 for i in range(n_roi):
     roi_pos[i, :] = [(np.real(positions[i]))/3, (np.imag(positions[i]))/3]
 
+# scale based on aspect ratio
+scale=[win.size[1]/win.size[0], 1]
```

Comment:
This part is also needed to rescale the circles/ball so that they are still circular despite the full computer window being rectangular and changing the aspect ratio